src/components/form/widgets/input-field.ts

```diff
@@ -71,19 +72,19 @@ function getInputType(schema: any): InputType {
     return 'text';
 }
 
-function getStepSize(schema: any): 'any' | number {
+function getStepSize(schema: FormSchema): 'any' | number {
     if (isNumberType(schema) && schema.multipleOf) {
-        return parseFloat(schema.multipleOf) || 'any';
+        return parseFloat(String(schema.multipleOf)) || 'any';
     }
 
     if (isIntegerType(schema)) {
-        return parseInt(schema.multipleOf, 10) || 1;
+        return parseInt(String(schema.multipleOf), 10) || 1;
```

Comment:
An alternative to `parseInt` and `parseFloat`, which I _usually_ recommend, is `+` (apparently called "unary plus" in this case). Here's a good explanation of the differences, copied from https://stackoverflow.com/a/17106701/280972:

---

Well, here are a few differences I know of:

- An empty string `""` evaluates to a `0`, while `parseInt` evaluates it to `NaN`. IMO, a blank string should be a `NaN`.

        +'' === 0;              //true
        isNaN(parseInt('',10)); //true

- The unary `+` acts more like `parseFloat` since it also accepts decimals. 

    `parseInt` on the other hand stops parsing when it sees a non-numerical character, like the period that is intended to be a decimal point `.`.

        +'2.3' === 2.3;           //true
        parseInt('2.3',10) === 2; //true

- `parseInt` and `parseFloat` parses and builds the string *left to right*. If they see an invalid character, it returns what has been parsed (if any) as a number, and `NaN` if none was parsed as a number. 

    The unary `+` on the other hand will return `NaN` if the entire string is non-convertible to a number.

        parseInt('2a',10) === 2; //true
        parseFloat('2a') === 2;  //true
        isNaN(+'2a');            //true

- As seen in the comment of [@Alex K.](https://stackoverflow.com/users/246342), `parseInt` and `parseFloat` will parse by character. This means hex and exponent notations will fail since the `x` and `e` are treated as non-numerical components (at least on base10).

    The unary `+` will convert them properly though.

        parseInt('2e3',10) === 2;  //true. This is supposed to be 2000
        +'2e3' === 2000;           //true. This one's correct.

        parseInt("0xf", 10) === 0; //true. This is supposed to be 15
        +'0xf' === 15;             //true. This one's correct.

---

I don't know what behaviour is the more appropriate for this use case, but generally, I would say that `+` seems to do the more correct thing in all cases except when the input is an empty string, and that case could easily be tested for first, and be made to return `NaN` instead.

For the `isIntegerType` case, we might also want to use `Math.floor` or `Math.round`…

Reply:
Oh, and, as an added bonus, you don't need to convert the value to a string before using unary plus 😊

And I realised I didn't show how it's used:

```suggestion
        return +schema.multipleOf || 1;
```

And I just now realised that we don't need to treat an empty string differently, as `0` is falsy, so it will return `1` instead 😄

Reply:
Fixed, did it on the `parseFloat` in a few lines above as well